Raksheeth/script/app.js

```diff
@@ -0,0 +1,6 @@
+var uname= document.getElementsByClassName("user-name")[0];
+var showname=uname.value;
+function updateContent() {
+  
+    document.write(showname);
```

Comment:
This wont work. It will always return blank value, which was supposedly the initial value of the input text.